src/models/hashmodel.h

```diff
@@ -104,6 +106,16 @@ class HashModel : public QAbstractTableModel
         return index(row, column);
     }
 
+    QModelIndex indexForSymbol(Data::Symbol symbol)
```

Comment:
m_rows is private, is there another way to get the rows from outside the HashModel without having to make m_rows protected? Should I make m_rows protected? Also what is SymbolHashModel?